regulations/generator/toc.py

```diff
@@ -102,12 +117,17 @@ def intro_interps(toc, reg_part):
     return elements
 
 
-def subterps(so_far, reg_part):
+def subterps(so_far, reg_part, version=None):
     """Logic to build subterps, collections of interpretations for subparts,
     the empty subpart, or appendices"""
+
     elements = []
     found_subpart = False
-    found_appendix = False
+    if version in IGNORE_APPENDIX_INTERPS:
+        found_appendix = True
+    else:
+        found_appendix = False
+    print so_far, reg_part
```

Comment:
This is merged - but here's a random print statement that probably should be removed?